src/components/publishing/__test__/imageset_preview.test.tsx

```diff
@@ -1,10 +1,14 @@
 import ImagesetPreview from "../imageset_preview"
-import { Images } from "./fixtures"
+import { ImageSetFull, ImageSetMini } from "./fixtures"
 
 import * as React from "react"
 import * as renderer from "react-test-renderer"
 
-it("renders properly", () => {
-  const imageset = renderer.create(<ImagesetPreview images={Images} />).toJSON()
+it("renders a full image-set properly", () => {
+  const imageset = renderer.create(<ImagesetPreview section={ImageSetFull} />).toJSON()
```

Comment:
Is ‘imageset’ 1 word? I would _think_ it’s 2 words ‘image-set’ and thus the fixture name is more correct than the component name. What say you?